### PR TITLE
Add missing sdk initializer

### PR DESCRIPTION
**Pull request by cab105**
Follow-up to #318  add missing AWS SDK client initialization codebase.

### CODE REVIEW

File: pkg/targets/adapter/awslambdatarget/adapter.go [hunk 1]
```diff
@@ -60,8 +62,7 @@ type adapter struct {
 	awsArnString string
 	awsArn       arn.ARN
 	config       *aws.Config
```
Comment (JeffNeff) [inline pkg/targets/adapter/awslambdatarget/adapter.go:64]:
```suggestion

```

File: pkg/targets/adapter/awssnstarget/adapter.go [hunk 2]
```diff
@@ -59,7 +63,7 @@ type adapter struct {
 	awsArn       arn.ARN
 	config       *aws.Config
```
Comment (JeffNeff) [inline pkg/targets/adapter/awssnstarget/adapter.go:64]:
```suggestion

```

File: pkg/targets/adapter/awssnstarget/adapter.go [hunk 3]
```diff
@@ -59,7 +63,7 @@ type adapter struct {
...
 	session      *session.Session
```
Comment (JeffNeff) [inline pkg/targets/adapter/awssnstarget/adapter.go:65]:
```suggestion

```

**Review by odacremolbap — APPROVED** [review]

**Review by odacremolbap — APPROVED** [review]
+1 when linter is happy

File: pkg/targets/adapter/awslambdatarget/adapter.go (hunk 1 above)
Comment (cab105) [reply pkg/targets/adapter/awslambdatarget/adapter.go]: Good catch!

File: pkg/targets/adapter/awssnstarget/adapter.go (hunk 3 above)
Comment (cab105) [reply pkg/targets/adapter/awssnstarget/adapter.go]: Another good catch!

**Pull request merged**